controllers/admin/AdminTranslationsController.php

```diff
@@ -83,7 +87,8 @@ public function __construct()
 
         parent::__construct();
 
-        $this->link_lang_pack = str_replace('%ps_version%', _PS_VERSION_, $this->link_lang_pack);
+        $language_pack_loader = RemoteLanguagePackLoader::build();
```

Comment:
We usually prefer camel case for new variables, the legacy code is full of snake case and we can't fix it all (it would not be worth investing the time), but when it's new code we might start on the right path (especially for local variables)